crates/re_viewer/src/ui/wait_screen_ui.rs

```diff
@@ -198,12 +198,11 @@ fn onboarding_content_ui(
             url_large_text_button(re_ui, ui, "Rust", RUST_QUICKSTART);
         });
 
-        #[cfg(not(target_arch = "wasm32"))]
         {
-            use re_ui::UICommandSender;
+            use re_ui::UICommandSender as _;
```

Comment:
Is that still needed?

Reply:
Yeah, the trait import is needed for `send_ui`. I added the `as _` because of https://github.com/rerun-io/rerun/blob/main/CODE_STYLE.md#style